src/openforms/registrations/contrib/email/templates/emails/email_registration.txt

```diff
@@ -1,4 +1,4 @@
-{% load i18n %}{% with form_name=renderer.submission.form.admin_name %}{% autoescape off %}{% if payment_received %}{% blocktrans trimmed %}
+{% load i18n form_summary %}{% with form_name=renderer.submission.form.admin_name %}{% autoescape off %}{% if payment_received %}{% blocktrans trimmed %}
```

Comment:
We dropped more builtins. Was this the form_summary the only one  that tripped a test? Or did we cover them already in the other PR?

Reply:
we only allow the builtins in sandbox templates because it's hard to document and understand for form builders that they have to `{% load %}` a library before they can use certain tags. That also limits our options to refactor/move these custom tags around.

These templates however are only under our control as developers, so we can use the 'normal' Django template operation and make things more explicit.